### PR TITLE
Including a Google Colab notebook

### PR DESCRIPTION
**Pull request by mezarque**
This PR does two things:

- it replaces the two Jupyter notebooks previously used for generating interactive plotly HTML files with a pair of new notebooks that generate the plot in a slightly different, cleaner way (notebooks starting with `4_DrerMmusXlae`)
- it adds the framework for loading the repo in Binder, although this loading currently fails due to Binder's size limitations (see: binder_notebook.ipynb, environment.yml, and postBuild)
- it includes a link to a [Google Colab notebook](https://colab.research.google.com/drive/1EVworofkY_-ClFfTmvHdjpLChFG4-sDP?usp=sharing) which implements the functionality of the desired Binder, since the amount of data and compute needed are too large.

The Colab notebook does the following things:
- downloads annotations for each of the three species' cell types (Drer, Mmus, Xlae) from [this Zenodo record](https://zenodo.org/record/7838976)
- downloads compressed .h5ad files for the joint embedding of cells from all three species in orthogroup or structural cluster space
- performs filtering using scanpy, species-based harmonization using Harmony, and dimensionality reduction using the same parameters as in the pub
- plots the UMAP of the joint embedding as colored by species, celltype, and Leiden cluster
- provides the data for downstream exploration using Scanpy

While not as tidy as Binder, hopefully this Colab notebook provides enough of an entry point to the data of this pub that it's useful.

The dramatically high number of lines changed is because of how the output of Plotly interacts with Jupyter notebooks; there's not actually a ton of code that has been modified.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.10.10
+  - anndata=0.8.0
+  - harmonypy
+  - matplotlib=3.5.3
+  - matplotlib-base=3.5.3
+  - numpy
```
Comment (taylorreiter) [inline environment.yml:11]: can you add a version here?

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - pandas=1.4.4
+  - plotly
```
Comment (taylorreiter) [inline environment.yml:13]: can you add a version here?

File: environment.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.10.10
+  - anndata=0.8.0
+  - harmonypy
```
Comment (taylorreiter) [inline environment.yml:8]: can you add a version here?

**Review by taylorreiter — APPROVED** [review]
Im going to go ahead an approve, but there are a couple changes that would be good!

also is `binder_notebook.ipynb` the same thing as the google colab notebook that's linked? they look a little different -- it would be good if the colab nb was the same as whatever is on github if possible.

but i ran the colab notebook and it all worked! very exciting stuff. so fun to dive in! v well organized too. I did notice you have two section 5s in that notebook, but that's not really a big deal.

**Comment by mezarque** [comment]
> also is `binder_notebook.ipynb` the same thing as the google colab notebook that's linked? they look a little different -- it would be good if the colab nb was the same as whatever is on github if possible.

The two notebooks have the same function, but they do have some minor differences:
- rather than using `conda` and the `environment.yml` that Binder would use, the Colab notebook uses `pip` to install software dependencies
- the Colab notebook also has a cell that downloads files using `wget` which would have been handled by `postBuild`
- I added a variety of form comments that allow Colab to pass in user-input variables through its GUI-like markdown format for people to modify parameters without having to directly edit the code

Hopefully it should be similar enough that it won't be confusing!

**Pull request merged**